docs/user/README.md

```diff
@@ -27,14 +27,13 @@ Within the API Gateway module, API Gateway Operator manages the application of A
 
 ### APIGateway Controller
 
-APIGateway Controller manages the installation of [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper) and handles the configuration of Kyma Gateway and the resources defined in the APIGateway custom resource (CR). The controller is responsible for:
-- Installing, upgrading, and uninstalling Ory Oathkeeper
+APIGateway Controller handles the configuration of Kyma Gateway and the resources defined in the APIGateway custom resource (CR). The controller is responsible for:
 - Configuring Kyma Gateway
 - Managing Certificate and DNSEntry resources
 
 ### APIRule Controller
 
-APIRule Controller uses [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper) and [Istio](https://istio.io/) resources to expose and secure APIs.
+APIRule Controller uses [Istio](https://istio.io/) resource to expose and secure APIs.
```

Comment:
```suggestion
APIRule Controller uses [Istio](https://istio.io/) resources to expose and secure APIs.
```
I guess it uses multiple resources, right? like AuthorizationPolicy, VirtualService, and RequestAuthentication